defaults.yaml

```diff
@@ -2,7 +2,7 @@ components:
   zss:
     port: 7557
     crossMemoryServerName: ${{ ()=> { if (zowe.environments?.ZWED_privilegedServerName) { return zowe.environments.ZWED_privilegedServerName } else { if (zowe.environments?.ZWES_XMEM_SERVER_NAME) { return zowe.environments.ZWES_XMEM_SERVER_NAME } else { return "ZWESIS_STD" } } }() }}
-    tls: true
+    tls: ${{ ()=> { return !(zowe.network.attls || components.zss.launcher.attls) }() }}
```

Comment:
its possible for network or launcher to not exist, do:
${{ ()=> { return !(zowe.network?.attls || components.zss.launcher?.attls) }() }}